src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1049,6 +1052,9 @@ theorem lt_bsup_of_limit {o : ordinal} {f : Π a < o, ordinal}
   (ho : o.is_limit) (i h) : f i h < bsup o f :=
 lt_of_lt_of_le (hf _ _ $ lt_succ_self i) (le_bsup f i.succ $ ho.2 _ h)
 
+theorem bsup_const {o : ordinal} (ho : 0 < o) (a : ordinal) : bsup o (λ _ _, a) = a :=
```

Comment:
Could this lemma take the assumption `o != 0` rather than `0 < o`, as in the mathlib guide?